src/stories/Library/Forms/input/DoubleInputRow.tsx

```diff
@@ -0,0 +1,30 @@
+import Input from "./Input";
+
+export type DoubleInputRowProps = {
+  leftLabel: string;
+  rightLabel: string;
+};
+
+const DoubleInputRow: React.FC<DoubleInputRowProps> = ({
+  leftLabel,
+  rightLabel,
+}) => {
+  return (
+    <div className="contact-info-flex">
```

Comment:
The naming and placemnt of this component indicates that it is a general form input component. But this class name is very specific. What are youre thoughts here

Reply:
Yesss, it's because we already have that class and use it in the V1 component. I could make a copy of it though, with the same properties if you think that works better :)

Reply:
I think we should go as generic as possible moving forward. So if it implies copying a component(s) and consider the old ones deprecated, I think we should do that. We can even write in the old ones that they are deprecated -> even name them `BlaBlaDeprecated.tsx`